CODE/Package.swift

```diff
@@ -17,10 +17,10 @@ let package = Package(
         .library(name: "PennyServices", targets: ["PennyServices"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", revision: "98a23b64bb5feadf43eed2fc1edf08817d8449b4"),
+        .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", revision: "c915322ecad44006790c72646380a897d3199342"),
         .package(url: "https://github.com/swift-server/swift-aws-lambda-events.git", branch: "main"),
         .package(url: "https://github.com/soto-project/soto.git", from: "6.2.0"),
-        .package(url: "https://github.com/mahdibm/DiscordBM.git", exact: "1.0.0-beta.29"),
+        .package(url: "https://github.com/mahdibm/DiscordBM.git", branch: "main"),
```

Comment:
i'll probably tag the new changes before push to main. or maybe pinning to a commit.